docs/source/demos/different_DB_names.rst

```diff
@@ -0,0 +1,149 @@
+.. _different_DB_names:
+
+============================================
+Running MOTECH with different database names
+============================================
+
+.. contents:: Table of Contents
+   :depth: 3
+
+Background
+==========
+One of our partners requested a tutorial on running MOTECH with different database names so they can run multiple MOTECH instances in `Docker <http://www.docker.com>`_ containers pointing to a single SQL database. MOTECH is not setup to run as a multi-tenant SaaS system. That is, we assume one MOTECH instance maps to one information owner. This architecture was chosen to retain a separation of information and access to serve multiple customers. 
+
+We created the capability to modify database names in MOTECH v0.28. This tutorial identifies the files and steps involved in changing the database names on Ubuntu 14.04. To be clear, MOTECH doesn't have the capability to create the database names before bootstrap, so we will proceed in the following three parts:
+
+- *Part 1*: Setup and run a MOTECH default instance, configure it as you see fit
+- *Part 2*: Copy the databases and .motech/config folder to create your default system backup
+- *Part 3*: Modify the database names and deploy the new configuration
+
+Part 1
+------
+The first step is to follow the :doc: `implementer's setup guide <../get_started/installing>`_ to create your default instance. You don't need to install any modules, but you'll need to make sure you complete the following steps:
+- When you complete the bootstrap form **make sure to choose Configuration Mode: File** 
+- After you click the verify button, you'll get an error about the web security activator not being able to start. That's because we need to modify some of the properties for file based configuration. Follow these steps to get everything configured correctly:
+
+.. code-block:: bash
+
+   #Stop the tomcat7 server
+   sudo service tomcat7 stop
+   #Create a file called motech-settings.properties that has the configuration variables
+   echo -e "system.language=en\nlogin.mode=repository" | sudo tee --append ~/.motech/config/motech-settings.properties
```

Comment:
I'm assuming that tomcat7 is installed as a service with permissions tomcat7:tomcat7 on the .motech folder. When I tried the command, it gave me permission denied without sudo and with 'sudo -u tomcat7', so I chose to add sudo then, 'chown' in the next line.